src/patterns/check-answers/index.md.njk

```diff
@@ -38,14 +38,14 @@ However, if your service asks users for longer or more detailed answers, a full
 
 ### Make the page easy to understand
 
-On your check answers page you should:
+On your check answers page, you should:
 
-- make it clear there’s a task to perform on the page
+- use the page title to tell the user what they need to do — otherwise, they might miss the 'submit' button at the end of the page
```

Comment:
@EoinShaughnessy Another thought (apols!), wondering if the 'page title' instruction is important enough to put above the bullet list (which is also looking a bit long anyway)?

I cannot put this in as a code suggestion, but I'm happy to make a commit to get this in!

> ### Make the page easy to understand
> 
>Use the page title to tell the user what they need to do — otherwise, they might miss the 'submit' button at the end of the page.
>
> On the page, you should also:

Reply:
That sounds really good! Thanks. No worries re commit, I'll make that now.